src/libYARP_OS/src/NestedContact.cpp

```diff
@@ -62,7 +62,7 @@ bool NestedContact::fromString(const ConstString& nFullName) {
         nodeName = fullName.substr(0, idx);
         nestedName = fullName.substr(idx+1, fullName.length());
         idx = nestedName.find('/');
-        if (idx==0) return true;
+        if (idx<=0) return true;
```

Comment:
As discussed, this is probably a false positive by coverity, string::find returns a size_t. Please drop this commit